### DISCUSSION TITLE
Way too jargonized and still doesn't perform

### DISCUSSION THREAD

**Discussion by Vin1001** [amazon-science/chronos-forecasting#51] [Show and tell]
I made a model from scratch, even that gives better performance than this. Takes 11s to train on just 260 x 1 data, compared to 0.2s of my derivative-based model. The staff also doesn't cooperate, pretending like they don't know how to code, when shared full code with them. Don't believe it? Try my model yourself, here's my repo: https://github.com/Vin1001/TSA

Had to predict Amazon's own stock price, to show how Chronos performs against my model (Hope now they don't deny their own stock prices)
![5a87aa81-4ffe-4e9b-99a4-f8f683c6bdd5](https://github.com/amazon-science/chronos-forecasting/assets/120188703/ef52bac5-5dea-4555-bc27-942f6d662f4f)
![8ba93bda-6788-4708-9d0e-312249252f1b](https://github.com/amazon-science/chronos-forecasting/assets/120188703/6ba8257f-e4d3-4bd0-8dc3-812409418793)

**Comment by canerturkmen** [comment]
Hi @Vin1001. Please refer to the [code of conduct](https://aws.github.io/code-of-conduct) for the rules on this forum. 

I believe @abdulfatir has made a commendable effort of pointing out the clear issues with your comparison in #49. I will now close this issue.
